aiven/resource_project.go

```diff
@@ -60,6 +67,42 @@ var aivenProjectSchema = map[string]*schema.Schema{
 		Elem:        &schema.Schema{Type: schema.TypeString},
 		Optional:    true,
 	},
+	"cloud": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Target cloud",
```

Comment:
`Default cloud for new services` would be more descriptive and perhaps the field name could be set to `default_cloud`.